### ISSUE TITLE
Wheel for version 1.6.7 causes 'Illegal instruction: 4' on OSX

### ISSUE THREAD

**Issue by miccoli** [rfk/pyenchant#85]
The binaries included within `pyenchant-1.6.7-py2.py3.cp27.cp32.cp33.cp34.cp35.pp27.pp33-none-macosx_10_6_intel.macosx_10_9_intel.whl` cause an `Illegal instruction: 4` error.

```
$ pyenv -v
pyenv 20160726
$ pip install pyenchant --no-cache-dir
Collecting pyenchant
  Downloading pyenchant-1.6.7-py2.py3.cp27.cp32.cp33.cp34.cp35.pp27.pp33-none-macosx_10_6_intel.macosx_10_9_intel.whl (4.8MB)
    100% |################################| 4.8MB 12.2MB/s 
Installing collected packages: pyenchant
Successfully installed pyenchant-1.6.7
$ python
Python 3.5.2 (default, Jul 30 2016, 16:05:06) 
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import enchant
Illegal instruction: 4
```

A quick work-around is 

```
$ pip uninstall -yq pyenchant
$ pip install --no-cache-dir --no-binary :all: pyenchant
Collecting pyenchant
  Downloading pyenchant-1.6.7.tar.gz (63kB)
    100% |################################| 71kB 1.9MB/s 
Installing collected packages: pyenchant
  Running setup.py install for pyenchant ... done
Successfully installed pyenchant-1.6.7
$ python
Python 3.5.2 (default, Jul 30 2016, 16:05:06) 
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import enchant
>>> 
```

This error could be linked to the fact that the the wheel is compiled with optimisations that generate code not supported on older CPUs, see this [thread](https://software.intel.com/en-us/forums/intel-fortran-compiler-for-linux-and-mac-os-x/topic/624902#)

**Comment by rfk** [comment]
Thanks @miccoli, I'll take a look at the linked thread.  What version of OS X are you testing on?

**Comment by miccoli** [comment]
The tests are performed on OS X 10.11.6. The CPU is reported as an `Intel(R) Core(TM) i7-3720QM CPU @ 2.60GHz`

**Comment by rfk** [comment]
In the meantime, I've removed the binary distribution from PyPI so that folks don't have to work around it.

**Comment by alex** [comment]
FYI, this shows up travis's OS X.

**Comment by rfk** [comment]
Thanks, will make testing a fixed version a little easier

**Comment by reaperhulk** [comment]
@rfk we've had this problem with pyca/cryptography wheels in the past when I forgot to rebuild the libraries we were statically linking out of homebrew with `--build-bottle` (which sets arch to core2duo and will work on any mac). I'm guessing you probably did a `brew reinstall enchant --universal` to get a universal build, but you really need `brew reinstall enchant --universal --build-bottle`

**Comment by rfk** [comment]
Thanks for the tip @reaperhulk; following your advice, I've managed to make a .whl that appears to work when run under the travis OS X environment:

  https://rfk.id.au/static/scratch/pyenchant-1.6.7-py2.py3.cp27.cp32.cp33.cp34.cp35.pp27.pp33-none-macosx_10_6_intel.macosx_10_9_intel.whl

@miccoli could you please try this version out and let me know if it works for you?  If so I'll make a 1.6.8 release to include it.

**Comment by miccoli** [comment]
82c35488c6566d6eaa32ba270b59cc1ad9cfc7f0  pyenchant-1.6.7-py2.py3.cp27.cp32.cp33.cp34.cp35.pp27.pp33-none-macosx_10_6_intel.macosx_10_9_intel.whl

works fine for me, thx.

**Comment by jhamrick** [comment]
Works for me too!

**Comment by rfk** [comment]
Alrighty, v1.6.8 released, thanks for all the help in figuring out the problem here!

**Comment by alex** [comment]
@rfk is there any chance this regressed with the `.9` and `.10` releases? We're seeing crashes on imports, though they're SIGSTOP, not illegal instruction: https://travis-ci.org/pyca/cryptography/builds/255439985#L848

**Comment by rfk** [comment]
> is there any chance this regressed with the .9 and .10 releases? 

Hrm, perhaps homebrew rebuilt these without the special `--build-bottle` as part of some larger upgrade.  Thanks for the heads-up, I'll see if I can push  fixed release later today.

**Comment by rfk** [comment]
OK, this should be fixed in the new 1.6.11 release, @alex let me know if you're still having issues.

**Comment by alex** [comment]
Confirmed that this apepars to have fixed it. Thanks!
